### ISSUE TITLE
Chronyd and PTP should cannot be enabled in the same node

### ISSUE THREAD

**Issue by williamcaban** [openshift-kni/cnf-features-deploy#522]
When using PTP Operator, it does contain the `phy2sys` which does time sync between PTP and the system. Having chronyd also active and doing clock sync, will be a conflict of two different processes (one with high time resolution other with regular time resolution) modifying the system clock. This affect workloads that rely on PTP and assume the system is also sync to PTP.

Remediation:
- For the SNO DU and the RWN DU, make sure to disable chronyd systemd service on the nodes

**Comment by ijolliffe** [comment]
@josephdrichard please have a look.

**Comment by yuvalk** [comment]
like this
https://github.com/openshift-kni/cnf-features-deploy/blob/master/feature-configs/5g-ref-nw/profile-base/du-general/ptp/disable-chronyd/disable-chronyd.yaml
?

I'd say though, we do miss an automated test for it.

**Comment by williamcaban** [comment]
Yes, like that and to make sure it is part of the ZTP policy generator

**Comment by vitus133** [comment]
@serngawy 
@imiller0 
Could you guys please add it to the pipeline manifests? The need for patch is going to stay here for a while. As I understand it's not going to be handled by the PTP operator because there is a case where PTP benefits from NTP,

**Comment by serngawy** [comment]
@vitus133 yes, we have it https://github.com/openshift-kni/cnf-features-deploy/blob/master/ztp/source-policy-crs/MachineConfigDisableChronyd.yaml 

I believe @imiller0 included it in the pipeline for SNO testing

**Comment by imiller0** [comment]
Yes. This policy is part of the DU profile that is applied to clusters deployed through our ZTP pipeline.

**Comment by imiller0** [comment]
Yesterday I added a validation in the ZTP metrics pipeline which checks that the chronyd machine config (disables chronyd) is applied to the active SNO MCP.

**Comment by ijolliffe** [comment]
Thanks all and @imiller0 for the updates - seems like we can close this issue.
